resource-managers/kubernetes/core/pom.xml

```diff
@@ -20,7 +20,7 @@
   <parent>
     <groupId>org.apache.spark</groupId>
     <artifactId>spark-parent_2.11</artifactId>
-    <version>2.2.0-SNAPSHOT</version>
+    <version>2.1.0</version>
```

Comment:
That's a good point. Do we switch over all the POMs to publish our version? Including say - sql, ml and other parts we did not touch?